nixos/modules/services/misc/homepage-dashboard.nix

```diff
@@ -247,12 +247,49 @@ in
 
         serviceConfig = {
           Type = "simple";
-          DynamicUser = true;
           EnvironmentFile = lib.mkIf (cfg.environmentFile != null) cfg.environmentFile;
           StateDirectory = lib.mkIf (!managedConfig) "homepage-dashboard";
           CacheDirectory = "homepage-dashboard";
           ExecStart = lib.getExe cfg.package;
           Restart = "on-failure";
+
+          # hardening
+          DynamicUser = true;
+          DevicePolicy = "closed";
+          CapabilityBoundingSet = "";
+          RestrictAddressFamilies = [
+            "AF_INET"
+            "AF_INET6"
+            "AF_UNIX"
+            "AF_NETLINK"
+          ];
+          DeviceAllow = "";
+          NoNewPrivileges = true;
+          PrivateDevices = true;
+          PrivateMounts = true;
+          PrivateTmp = true;
+          PrivateUsers = true;
+          ProtectClock = true;
+          ProtectControlGroups = true;
+          ProtectHome = true;
+          ProtectKernelLogs = true;
+          ProtectKernelModules = true;
+          ProtectKernelTunables = true;
+          ProtectSystem = "strict";
+          LockPersonality = true;
+          RemoveIPC = true;
+          RestrictNamespaces = true;
+          RestrictRealtime = true;
+          RestrictSUIDSGID = true;
+          SystemCallArchitectures = "native";
+          SystemCallFilter = [
+            "@system-service"
+            "~@resources"
+          ];
+          ProtectProc = "invisible";
+          ProtectHostname = true;
+          ProcSubset = "pid";
```

Comment:
Setting this to `pid` breaks `services.homepage-dashboard.widgets = [{ resources.cpu = true; }];`, always showing 0% usage.

Reply:
Thank you for catching that!  I updated the module.

I tested with the other resource types, everything else worked: https://gethomepage.dev/widgets/info/resources/